data/application.css

```diff
@@ -121,6 +121,18 @@ image.contact-default-image {
   color: #659e51;
 }
 
+.input-bubble-container .chatbox-bubble-contents.input .overlay-label {
+  opacity: 0.0;
+  margin-left: 10px;
+  font-style: italic;
+  transition: opacity 0.15s, margin-left 0.15s;
```

Comment:
@erikos @cosimoc I added this small animation here so that the text fades and slides out/in from the right. Its very subtle but I think it actually works quite well, let me know what you think.